tests/DistributedTracing/DistributedTraceTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace DDTrace\Tests\DistributedTracing;
+
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\RequestSpec;
+
+class DistributedTraceTest extends WebFrameworkTestCase
+{
+    protected static function getAppIndexScript()
+    {
+        return __DIR__ . '/../Frameworks/Custom/Version_Not_Autoloaded/index.php';
+    }
+
+    protected static function getEnvs()
+    {
+        return array_merge(parent::getEnvs(), [
+            'DD_DISTRIBUTED_TRACING' => 'true',
+            'DD_TRACE_NO_AUTOLOADER' => 'true',
+        ]);
+    }
+
+    public function testDistributedTrace()
+    {
+        $traces = $this->tracesFromWebRequest(function () {
+            $spec = new RequestSpec(
+                __FUNCTION__,
+                'GET',
+                '/index.php',
+                [
+                    'x-datadog-trace-id: 1042',
+                    'x-datadog-parent-id: 1000',
+                ]
+            );
+            $this->call($spec);
+        });
+
+        $this->assertSame(1042, $traces[0][0]['trace_id']);
+        $this->assertSame(1000, $traces[0][0]['parent_id']);
+    }
+
+    // Synthetics requests have "0" as the parent ID
+    public function testDistributedTraceWithNoParent()
+    {
+        $traces = $this->tracesFromWebRequest(function () {
+            $spec = new RequestSpec(
+                __FUNCTION__,
+                'GET',
+                '/index.php',
+                [
+                    'x-datadog-trace-id: 6017420907356617206',
```

Comment:
Why this specific value? I assume it will overflow a 64 bit signed int but other than that it looks arbitrary.

Reply:
It doesn't overflow, it's just a real ID generated by Synthetics I captured during a Synthetics test. :)